validator/weight_setting/validator.py

```diff
@@ -175,9 +174,7 @@ def __init__(self):
         self.contest = find_contest(self.contest_state.id) if self.contest_state else CURRENT_CONTEST
 
         self.websocket = self.connect_to_api()
-
-        self.log_thread = Thread(target=self.api_logs)
-        self.log_thread.start()
+        asyncio.run(self.api_logs())
```

Comment:
Why should this be on the async event loop?

Reply:
was having issues with the original thread solution but tried again and seems to be working now, will revert it